src/views/Setup.vue

```diff
@@ -89,7 +89,7 @@ export default {
       } else if (percentage <= 70) {
         return 'Running wallet'
       } else if (percentage > 70) {
-        return 'We are preparing Sheikah'
+        return 'Sheikah is ready'
```

Comment:
Now that you can try it, could you update percetanges as I mentioned [here](https://github.com/witnet/sheikah/pull/1316#discussion_r483624162)